nilearn/interfaces/fmriprep/load_confounds_utils.py

```diff
@@ -90,46 +90,74 @@ def _add_suffix(params, model):
 
 def _get_file_name(nii_file):
     """Construct the raw confound file name from processed functional data."""
```

Comment:
```suggestion
    """Construct the confound file name from processed functional data."""
```

Reply:
Good catch. I rewrote the whole summary, but thank you for noting the incorrect description.